floodviz/tests/test_peak_flow_utils.py

```diff
@@ -2,235 +2,245 @@
+                             'USGS	05481950	1975-03-18		2500	2	11.61	1\n' \
+                             'USGS	05481950	1976-04-18		1960		9.62\n' \
+                             'USGS	05481950	1977-08-28		852		7.81\n' \
+                             'USGS	05481950	1978-03-21		2980		11.38\n' \
+                             'USGS	05481950	1979-03-20		4700		13.88\n' \
+                             'USGS	05481950	1980-06-04		1730		9.02\n' \
+                             'USGS	05481950	1981-08-14		1140		7.71\n' \
+                             'USGS	05481950	1982-05-28		2780		11.32\n' \
+                             'USGS	05481950	1983-06-29		3130		12.41\n' \
+                             'USGS	05481950	1984-05-02		3220		11.76\n' \
+                             'USGS	05481950	1984-12-28		1120		7.66\n' \
+                             'USGS	05481950	1986-06-30		7980		14.73\n' \
+                             'USGS	05481950	1987-08-28		2610		11.50\n' \
+                             'USGS	05481950	1987-11-30		544		6.34	2		1988-02-01		6.91	1\n' \
+                             'USGS	05481950	1989-02-01		1540	2	11.35	1\n' \
+                             'USGS	05481950	1990-06-19		5580		14.20\n' \
+                             'USGS	05481950	1991-06-16		3340		11.96\n' \
+                             'USGS	05481950	1992-04-23		1790		9.76\n' \
+                             'USGS	05481950	1993-07-10		14300		16.58\n' \
+                             'USGS	05481950	1994-03-06		1580		9.24\n' \
+                             'USGS	05481950	1995-05-30		1960		10.12\n' \
+                             'USGS	05481950	1996-05-10		1410		8.88\n' \
+                             'USGS	05481950	1997-07-24		1920		9.97	2		1997-02-20		10.50	1\n' \
+                             'USGS	05481950	1998-06-17		4280		13.20\n' \
+                             'USGS	05481950	1999-06-12		3820		12.36\n' \
+                             'USGS	05481950	2000-07-05		533		6.36\n' \
+                             'USGS	05481950	2001-03-17		2350		10.90\n' \
+                             'USGS	05481950	2002-05-30		912		7.55	2		2002-05-13		7.70\n' \
+                             'USGS	05481950	2003-05-06		3160		11.99\n' \
+                             'USGS	05481950	2004-05-24		7300		14.59\n' \
+                             'USGS	05481950	2005-05-14		2530		11.19\n' \
+                             'USGS	05481950	2006-09-14	04:30	1740		9.47\n' \
+                             'USGS	05481950	2007-02-02		850		13.86\n' \
+                             'USGS	05481950	2007-04-27		6850		13.86\n' \
+                             'USGS	05481950	2008-06-01		7800		14.51'
+
+        self.correct_outupt = self.correct_historic_peaks_output
+
+        self.prefix = 'http://example.com/'
+        self.end_date = '2008-07-05'
+        self.site = '05481950'
+        self.mock_url = self.prefix + '?site_no=' + self.site + '&agency_cd=USGS&format=rdb&end_date=' + self.end_date
+
+    def test_bad_status_code(self):
```

Comment:
To create a true unit test, you would not be mocking something that is called by a function that is called by the unit under test. Instead you would mock utils.parse_rdb. You can leave this as is. But something to consider for the future.